Modules/Module_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -58,13 +57,12 @@
 		
         x = xASL_imp_NII2BIDS_Run(x, bidsPar, studyParSpecificSubjVisitSess, listRuns, nameSubjectSession, bidsLabel, iRun);
     end
-    
 end
 
-
-
+%% -------------------------------------------------------------
 %% Check if there is a visit alias within the subject/visit name
-function bidsLabel = xASL_imp_CheckForAliasInVisit(imPar,nameSubjectSession)
+%% -------------------------------------------------------------
+function bidsLabel = xASL_imp_CheckForAliasInVisit(imPar, nameSubjectSession)
```

Comment:
Again confusing. 

So at line 58 above, it is clear SubjVisitSess (which means SubjSessRun).
But here the subfunction name is in legacy and the input variable name is in BIDS? or legacy?

I don't care what we choose, as long as we choose 1 nomenclature per function :)
Or we fix that in a follow-up issue, if you have tested thoroughly that we're not mixing up visits and sessions. I would fix the easy things now already, we don't want to mix things up.

Reply:
All renamed to session.

But tokenVisitAlias and things inside imPar just have to stay visits...